.github/workflows/build-and-test.yaml

```diff
@@ -13,6 +13,7 @@ jobs:
   # it's hard to setup it in the container.
   download-test-data:
     runs-on: ubuntu-latest
+    timeout-minutes: 20
```

Comment:
Why not adding this statement in `build-and-test`?

Reply:
Thanks for letting me know, I added the statement in https://github.com/tier4/tier4_perception_dataset/pull/89/commits/9cb2a0c56b885e5e398b36678563442bbbbb8bb9